\label{diagnostic}
\begin{DoubleSpace*}
\section{Acute Myocardial Infarction}
Acute Myocardial Infarction (AMI), which is an acute manifestation of coronary heart disease, is defined by myocardial ischemia (exposure of cardiac myocytes to oxygen deprivation) and necrosis (a form of cell death) \cite{trainor2018}. AMI may occur following atherosclerotic plaque disruption or other conditions which cause demand ischemia \cite{thygesen2012,arbab2012}. Irrespective of underlying cause, ischemia and necrosis are the common pathological characteristics of all AMI. Thrombotic MI (MI results from spontaneous atherosclerotic plaque disruption with the formation of an occluding coronary thrombus) versus non-thrombotic MI represents an important etiological distinction \cite{defilippis2017}, as both types necessitate different treatment approaches \cite{thygesen2012}. A diagnosis of AMI can be substantiated by blood-based diagnostic tests that measure isoforms of the protein troponin which is released into the circulation following necrosis \cite{newby2012}. To date a blood-based diagnostic test capable of discriminating between thrombotic and non-thrombotic MI has not been developed, although it has been previously shown that a metabolic signature may differentiate between the types \cite{defilippis2017,trainor2018}. In the present work, we set out to develop a Bayesian model for differentiating thrombotic MI from both non-thrombotic MI and stable coronary artery disease (CAD) using metabolites detected in blood plasma. We regard plasma as a promising media for developing a non-invasive test as plasma contains hormones, enzymes, lipoproteins, and other metabolic intermediates found in circulation. As metabolite concentrations are a product of genetic factors, environmental exposures, and the interaction between the two, sampling metabolites may provide a more robust characterization of the state of an organism than other approaches such as genomics. 

\section{Clinical cohort and samples}
Towards the effort of developing a Bayesian diagnostic model, we utilized previously collected samples from a patient cohort that was recruited specifically for contrasting thrombotic MI from multiple control phenotypes \cite{defilippis2015,defilippis2017}. This cohort was comprised of three phenotypic groups of human subjects: thrombotic MI, non-thrombotic MI, and stable CAD. In reference to the  thrombotic MI group, both control groups (non-thrombotic MI and stable CAD) served as procedural controls as all groups underwent a cardiac catheterization procedure. The stable CAD group provides a stable disease control as both thrombotic MI subjects and stable CAD subjects have underlying coronary artery disease. Non-thrombotic MI subjects presented with myocardial necrosis and thus the non-thrombotic MI group serves as an acute disease event control. 

Whole blood was collected shortly before cardiac catheterization from all study subjects. Details of the sample handling, sample processing, separation by liquid or gas chromatography, and mass spectrometry analysis for quantifying metabolites are provided in Chapter~\ref{hdInteractome} Section~\ref{plasma}. 

\section{Feature selection}
A significant analytical challenge in developing a blood-based diagnostic test for differentiating MI types is to determine a small set of metabolites that should be included in the statistical model from a limited number of training samples. Specifically, 1,032 chemical features (identified compounds or unknown compounds) were detected from 11 thrombotic MI, 12 non-thrombotic MI, and 15 stable CAD subjects. To ensure that the statistical model is estimable from a small sample size, and as building a targeted MRM assay or multiplexed ELISA assay can only accommodate a small number of compounds, feature selection is a critical task. While other dimension reduction techniques such as latent variable approaches (e.g. Partial Least Squares models) that create new variables which are linear combinations of metabolites would be amenable for reducing the number of coefficients to be estimated in a classification model, these approaches would not reduce the number of metabolites needing to be quantified by future targeted assays. Consequently, we prioritized reducing the number of metabolites considered. To determine a statistical classifier with five metabolites, $9.7 \times 10^{12}$ combinations are possible. In order to search the space of possible models we employed a feature selection technique that utilizes an evolutionary algorithm and seeks a consensus solution over bootstrapped datasets as described in Trainor et al. \cite{trainor2018}. In this work, small sets of metabolites were included in a multinomial logit classifier. Each model represented an individual in a population. Genetic fitness was determined as the likelihood of each individual model. These populations of models were allowed to evolve given evolutionarily inspired processes such as birth, recombination, and death. Populations were grown over bootstrapped datasets to increase diversity and reduce the correlation between models in the populations. Finally, the frequency that an individual metabolite was included in models within the final population after epochs of evolution was determined yielding a variable importance score for each metabolite. A correlation plot illustrating the metabolites with greatest variable importance score using the described technique is shown in Figure~\ref{fig:include}.

\end{DoubleSpace*}

\begin{figure}[H]
	\resizebox{1.1\textwidth}{!}{\includegraphics*{../../AthroMetab/WoAC/corrplot333}}
	\caption[Correlation plot showing the metabolites that were considered in Bayesian multinomial logistic regression models]{\DoubleSpacing Metabolites with the highest variable importance score given the feature selection method we employed in a previous work \cite{trainor2018}. The Pearson correlation coefficients between metabolite transformed and scaled abundances are shown. \label{fig:include}}
\end{figure}

\begin{DoubleSpace*}
\section{Multinomial logistic regression model}
A multinomial logistic regression model was assumed for determining the  probability a sample from a clinical subject was member of the thrombotic MI, non-thrombotic MI, or a stable CAD study group. The multinomial logistic regression model is a generalized linear model and has the following link function and model form: 
\begin{align}
\eta_{ij} = \log \frac{\pi_{ij}}{\pi_{iJ}} = \alpha_j + \textbf{x}_i^T \boldsymbol{\beta}_j,
\end{align} 
where $i$ indexes individual samples, $j$ is the index of study groups, $\boldsymbol{\beta}_j$ is a vector of regression coefficients (with separate coefficients for each group), $\alpha_j$ is a group specific intercept term, and $J$ represents the reference group. Given a specific value for the link function, the probability a sample belongs to specific study group can be computed as:
\begin{align}
\hat{\pi}_{ij} = \frac{\exp \hat{\eta}_{ij}}{\sum_{k=1}^{J}\exp \hat{\eta}_{ik}}.
\end{align}

The model can be stated as a Bayesian model with the following priors and deterministic component:
\begin{align}
\begin{split}
	\alpha_j \sim N(0,4) \\
	\beta_j \sim N(0,1) \\
	\log \frac{\pi_{ij}}{\pi_{iJ}} = \alpha_j + \textbf{x}_i^T \boldsymbol{\beta}_j \\
	Y \sim Multinom(\boldsymbol{\pi})
\end{split}
\end{align}

\section{MCMC sampling of the posterior distribution}
A MCMC sampler known as the ``No-U-Turn'' sampler was utilized to simulate the posterior distribution of model parameters \cite{hoffman2014}. This algorithm is an extension of the Hamiltonian Monte Carlo algorithm. The Hamiltonian Monte Carlo algorithm is designed to model a target probability distribution as a Hamiltonian system \cite{betancourt2017}. In such a system, a parameter vector $\boldsymbol{\theta}$ is viewed as a particle in a $D$-dimensional space \cite{neal2011} by defining a vector field that is aligned with the typical set (the region of a parameter space with both significant volume and desnity) \cite{betancourt2017}. Defining such a vector field is a complex task. While a vector field could be defined using the gradient of the target probability distribution, this vector field would pull a particle towards the mode of the distribution. By adding a momentum term, the vector field can be defined so as to restrict a particle to maintaining the Hamiltonian at a fixed value. The Hamiltonian Monte Carlo algorithm utilizes Metropolis-Hastings proposals for updating both the momentum and position variables \cite{neal2011,betancourt2017}. A critical aspect of Hamiltonian Monte Carlo sampling is determining the optimal integration time for a particle to travel along a Hamiltonian path. One approach to dynamically determining this parameter is the ``No-U-Turn'' termination criteria \cite{hoffman2014,betancourt2017}. Conceptually, this criteria ensures that expansion of a trajectory continues to visit previously unexplored neighborhoods while terminating the trajectory when it returns to previously explored neighborhoods.

The ``No-U-Turn'' Hamiltonian MCMC sampler was implemented by others in \emph{Stan}, a probabilistic programming language \cite{carpenter2017}. Stan is written in C++ and provides the ``No-U-Turn'' sampler, a Hamiltonian Monte Carlo Sampler, a variational inference algorithm, and multiple optimizers. Stan provides grammar and syntax for succinctly specifying Bayesian models; can design a sampler for the posterior distribution of model parameters; and using a C++ compiler, Stan compiles the sampler to byte code. Additionally, Stan provides data structures for storing MCMC chains and model output. An R to Stan interface, \emph{rstan} \cite{stan2018}, has been developed allowing an end user to pass datasets between Stan and to return MCMC chains and model results. 

\section{Results}
In order to simulate the posterior distribution of model parameters, four MCMC chains were generated utilizing the No-U-Turn algorithm. An example MCMC chain is illustrated in Figure~\ref{fig:brm1Coef}(a). In this figure 2,000 of the 10,000 iterations of the regression coefficient parameter for 3-hydroxypyridine sulfate from one of the MCMC chains (chain \#1) is shown. Additionally, this figure shows how the posterior distribution is generated from the MCMC chain. In Figure~\ref{fig:brm1Coef}(b), by-group parameter histograms are shown for the multinomial logit model parameters corresponding to  3-hydroxypyridine sulfate. For this metabolite, the probability mass of posterior coefficient estimates is centered slightly below zero for non-thrombotic MI, while the center of the posterior distribution is above zero for thrombotic MI. From the MCMC simulated posterior distribution, Bayesian credible intervals were determined for each model parameter as shown in Figure~\ref{fig:brm1Coef}. 

As the objective of the present analysis was to develop a model capable of discriminating between the three phenotypes utilizing only the information available at the presentation of patients and in a non-invasive manner, we also considered introducing clinical troponin values. Bayesian credible intervals are presented from the posterior distribution given the introduction of troponin in Figure ~\ref{fig:brm2Coef}. Substantial qualitative changes in the credible intervals were not observed following the introduction of troponin. To compare the models, the ``Widely applicable information criterion'' was utilized (see Table~\ref{tab:modelComp}). A $\chi^2$ test on one degree of freedom reveals that troponin did not lead to a significant improvement overall ($p=0.36$). 

\begin{table}[H]
	\centering
	 	\caption[Bayesian multinomial logistic regression model goodness of fit showing the model, the Widely Applicable Information Criteria (WAIC), and the Standard Error (SE)]{\DoubleSpacing Evaluation of goodness of model fit with or without clinical troponin values. Inclusion of troponin did not result in a statistically meaningful improvement in goodness of fit. WAIC: Widely applicable information criteria. SE: Standard error.}
	 	\label{tab:modelComp}
	\begin{tabular}{lcc}
		\hline
       Model & WAIC  & SE \\
       \hline
       Without troponin & 15.43 & 4.05 \\ 
       With troponin & 14.60 & 3.72 \\
       Difference & 0.83 & 0.87 \\
       \hline
	\end{tabular}
\end{table}

\end{DoubleSpace*}

\begin{landscape}
\begin{figure}[H]
	\includegraphics[scale=.8]{../Aim3/MCMCEx.png}
	\caption[Time series plot of the MCMC chain for 3-hydroxypyridine sulfate multinomial logistic regression coefficient and resulting histogram showing the posterior distribution]{(a) MCMC chains for the by-group multinomial logit model parameters corresponding to 3-hydroxypyridine sulfate. (b) Histogram showing the simulated posterior probability distribution for the same parameters. \label{fig:brm1Coef} }
\end{figure}
\newpage
\begin{figure}[H]
	\includegraphics[scale=1.15]{../Aim3/brm1Coef}
	\caption[Bayesian credible intervals for multinomial logit model parameters without troponin]{\DoubleSpacing 50\% Bayesian credible intervals for the by-group multinomial logistic regression coefficients. The multinomial models did not include clinical troponin assay values. \label{fig:brm1Coef} }
\end{figure}
\newpage
\begin{figure}[H]
	\includegraphics[scale=1.15]{../Aim3/brm2Coef}
	\caption[Bayesian credible intervals for multinomial logit model parameters with troponin]{\DoubleSpacing 50\% Bayesian credible intervals for the by-group multinomial logistic regression coefficients. The multinomial models did include clinical troponin assay values. \label{fig:brm2Coef} }
\end{figure}
\end{landscape}

\begin{DoubleSpace*}
Given that the MCMC chains provide the joint posterior distribution of model parameters, the correlation between parameter estimates can be evaluated. Two parameter estimates were significantly correlated, both of which were for monoacylglycerols [1-linoleoylglycerol (18:2)] and [2-linoleoylglycerol (18:2)]. This relationship was further explored. Figure~\ref{fig:coefPost} shows a 3-dimensional scatterplot of samples from the simulated posterior distribution. In the first plane, the parameter estimates for both monoacylglycerols are shown, while the remaining axis shows the log-posterior probability of the model with these parameterizations.  Similarly, Figure~\ref{fig:coefPost2} shows the same slice of the posterior distribution with the log-posterior probability axis removed and represented as a color instead. The relatively strong negative correlation between the parameter estimates, without a systemic change in the log-posterior probability suggests that these two monoacylglycerols modulate the probability of the phenotypes in a similar way. 

\begin{figure}[H]
	\resizebox{1.0\textwidth}{!}{\includegraphics*{../Aim3/coefPost}}
	\caption[3-dimensional scatterplot of MCMC samples for both linoleoylglycerols (18:2) with the acyl side chain in different positions and the log-posterior probability of each sampled model]{\DoubleSpacing 3-dimensional scatterplot of MCMC samples for both linoleoylglycerols (18:2) with the acyl side chain in different positions (in the same plane), as well as the log-posterior probability of each sampled model. \label{fig:coefPost} }
\end{figure}

\begin{figure}[H]
	\resizebox{1.1\textwidth}{!}{\includegraphics*{../Aim3/coefPost2}}
	\caption[Scatterplot of MCMC samples for both linoleoylglycerols (18:2) with the acyl side chain in different positions colored by log-posterior probability]{\DoubleSpacing Scatterplot of MCMC samples for both linoleoylglycerols (18:2) with the acyl side chain in different positions colored by log-posterior probability. MCMC sampled models with the greatest log-posterior probability are shown in red, while models with lesser log-posterior probability are shown in blue. \label{fig:coefPost2} }
\end{figure}

From the simulated posterior distribution of models, a predictive phenotype probability distribution can be generated for each human subject. The MCMC chain for the phenotype probabilities of a selected clinical subject (\#14) is presented in Figure~\ref{fig:ptid2010MCMC}. The maximum \emph{a posteriori}) group label from both models evaluated properly classified this subject as a stable CAD subject, however in the leave-one-out cross-validation estimation, this subject misclassified as a Thrombotic MI subject. In the models (not-LOOCV estimates), a substantial probability mass was also placed on the posterior probability that this subject was a Non-Thrombotic MI.  

\begin{figure}[H]
	\resizebox{1.15\textwidth}{!}{\includegraphics*{../Aim3/ptid2010MCMC}}
	\caption[A segment of the first MCMC chain for simulating the posterior distribution of group membership probability estimates for a specific human subject. ]{\DoubleSpacing A segment of the first MCMC chain for simulating the posterior distribution of group membership probability estimates for a specific human subject.  \label{fig:ptid2010MCMC} }
\end{figure}

\begin{figure}[H]
	\resizebox{1.15\textwidth}{!}{\includegraphics*{../Aim3/ptid2010Hist}}
	\caption[Histograms showing the simulated posterior distribution of group membership probability estimates for a specific human subject]{\DoubleSpacing Histograms showing the simulated posterior distribution of group membership probability estimates for a specific human subject. \label{fig:ptid2010Hist} }
\end{figure}
 
The LOOCV-estimated confusion matrix is shown in Table~\ref{tab:modelRes0} for the model fit by maximum likelihood estimation (MLE), Table~\ref{tab:modelRes} for the Bayesian model without clinical troponin values, and Table ~\ref{tab:modelRes2} for the Bayesian model with clinical troponin values. For the Bayesian models, group membership was determined by maximum \emph{a posteriori} estimates. Finally, the raw probability estimates for each human subject from the Bayesian models are shown in Table~\ref{tab:modelRes3}. The LOOCV-estimated sensitivity for detecting and discriminating thrombotic MI was 90.9\% for the Bayesian models with and without clinical troponin values. The sensitivity for the model fit by MLE was 81.8\%. Without clinical troponin values, the estimated specificity for the Bayesian model was 96.3\%, while with clinical troponin values included the estimated specificity improved to 100.0\%. The model fit by MLE without clinical troponin values had a specificity of 92.6\%. In terms of the detection and discrimination of non-thrombotic MI, the LOOCV-estimated sensitivity was 83.3\% for both Bayesian models, and the estimated specificity was 88.5\% for both. In terms of the misclassification rate the model estimated by MLE had a rate of 21.1\%, while both Bayesian models had a  rate of 13.2\%. Two stable CAD subjects were classified by both Bayesian models as non-thrombotic MI subjects. Both of these subjects had relatively small estimated probabilities for thrombotic MI. Two non-thrombotic MI subjects were misclassified by both models. The first (subject \#26) was classified as stable CAD by both models. The second subject (\#27) was first classified as thrombotic MI (model without clinical troponin values), and then classified as stable CAD (model with clinical troponin values). This subject appeared to have nearly equal ($\approx 30\%$) probability mass placed over all three study groups. Finally, one thrombotic MI subject was classified as non-thrombotic MI by both models, however the difference between probability mass for thrombotic MI and non-thrombotic MI was slight ($46.3\%$ vs $39.0\%$ in the model without clinical troponin values; $44.8\%$ vs $36.0\%$ for the model with clinical troponin values).

\begin{table}[H]
 	\caption[Leave-one-out Cross-validation (LOOCV) estimated confusion matrix for the multinomial logistic regression model without clinical troponin values fit by Maximum Likelihood Estimation]{\DoubleSpacing Leave-one-out Cross-validation (LOOCV) estimated confusion matrix for the multinomial logistic regression model without clinical troponin values fit by Maximum Likelihood Estimation (MLE). The leftmost column shows the true group membership of the human subjects, while the remaining columns show the predicted probabilities (maximum \emph{a posteriori}).}
\label{tab:modelRes0}
\begin{tabular}{l|ccc}
& Predicted & & \\
Group  &     sCAD & Thrombotic MI & Non-Thrombotic MI \\
\hline
sCAD & 13 &  0 & 2\\
Thrombotic MI &   1 & 9 &  1\\
Non-Thrombotic MI  & 2  & 2 & 8 
\end{tabular}
\end{table}

 \begin{table}[H]
\centering
\caption[Leave-one-out Cross-validation (LOOCV) estimated confusion matrix for Bayesian multinomial logistic regression model without clinical troponin values]{\DoubleSpacing Leave-one-out Cross-validation (LOOCV) estimated confusion matrix for Bayesian multinomial logistic regression model without clinical troponin values. The leftmost column shows the true group membership of the human subjects, while the remaining columns show the predicted probabilities (maximum \emph{a posteriori}).}
\label{tab:modelRes}
\begin{tabular}{l|ccc}
& Predicted & & \\
Group  &     sCAD & Thrombotic MI & Non-Thrombotic MI \\
\hline
sCAD   &     13    &     0    &     2\\
Thrombotic MI  &  0    &    10    &     1\\
Non-Thrombotic MI &   1    &     1   &     10
\end{tabular}
 \end{table}
 
  \begin{table}[H]
  	\centering
  	\caption[Leave-one-out Cross-validation (LOOCV) estimated confusion matrix for Bayesian multinomial logistic regression model with clinical troponin values]{\DoubleSpacing Leave-one-out Cross-validation (LOOCV) estimated confusion matrix for Bayesian multinomial logistic regression model with clinical troponin values. The leftmost column shows the true group membership of the human subjects, while the remaining columns show the predicted probabilities (maximum \emph{a posteriori}).}
  	\label{tab:modelRes2}
\begin{tabular}{l|ccc}
& Predicted & & \\
Group  &     sCAD & Thrombotic MI & Non-Thrombotic MI \\
\hline
sCAD   &     13    &     0    &     2\\
Thrombotic MI  &  0    &    10    &     1\\
Non-Thrombotic MI &   2  &   0 &    10
\end{tabular}
  \end{table}
  
\begin{table}[H]
  	\centering
 	  	\caption[Leave-one-out cross-validation (LOOCV) estimated group probabilities from the Bayesian multinomial logistic regression models for each of human subject in the cohort.]{\DoubleSpacing  Leave-one-out cross-validation (LOOCV) estimated group probabilities from the Bayesian multinomial logistic regression models for each of human subject in the cohort. M1: Model without clinical troponin values. M2: Model with clinical troponin values. Blue-to-red continuous scale ranks estimated probabilities from low-to-high.}
	  	\label{tab:modelRes3}
 	\includegraphics[scale=.8]{../Aim3/PatientPred}
\end{table}

\section{Discussion}
The most notable conclusion from the construction of a Bayesian multinomial logistic regression model is that there is strong evidence that thrombotic MI, non-thrombotic MI, and stable CAD can be discriminated by a non-invasive method using a small set of circulating metabolites. High estimated sensitivity and specificity was observed for detecting and discriminating thrombotic MI from both non-thrombotic MI and stable CAD. The estimated sensitivity and specificity for detecting and discriminating non-thrombotic MI was lower for non-thrombotic MI. Given the heterogeneity of this etiology of acute MI, this observation is not surprising. 

There are many advantages from utilizing a Bayesian model estimation as in this context. First, given a relatively small sample size, maximum likelihood estimation for the multinomial logistic model may not allow for convex optimization. The specification of prior distributions with significant probability mass near zero for regression coefficients allows for the encouragement of sparsity. In addition, complete or quasi-complete separation of classes can occur utilizing maximum likelihood estimation, leading to unstable estimates for regression coefficients. As before, by specifying prior distributions for regression parameters that place significant probability mass near zero, the problem of complete or quasi-separation can be avoided. In general, a Bayesian model with appropriate priors will ensure that the resulting model has not overfit the training data with poor generalizability. A benefit of the Bayesian regression paradigm is the treatment of regression coefficients as random variables. In the Bayesian paradigm, variables that may contribute to the separation of groups, but for which there is much uncertainty can be retained in the model without adversely impacting model performance. Variables with correlated parameter estimates can be determined via the interrogation of samples from the posterior distribution of model coefficients. In our case, the regression coefficients for 1-linoleoylglycerol (18:2) and 2-linoleoylglycerol (18:2) were observed to be highly correlated in MCMC samples. This is not surprising given that these metabolites have the same glycerol moiety, and the same acyl side chain (although it is in different positions). Not surprisingly, the high degree of structural similarity is accompanied by a high degree of correlation in the abundance of each metabolite, and in the estimates of multinomial logistic regression parameters for each.

The frequentist model estimated by MLE exhibited inferior performance than either Bayesian model with respect to misclassification rate / accuracy, sensitivity, and specificity. 

\end{DoubleSpace*}