\begin{DoubleSpace*}
\section{Acute Myocardial Infarction}
Acute Myocardial Infarction (AMI), which is an acute manifestation of coronary heart disease, is defined by
myocardial ischemia (exposure of cardiac myocytes to oxygen deprivation) and necrosis (a form of cell death) \cite{trainor2018}. AMI may occur following atherosclerotic plaque disruption or other conditions which cause demand ischemia \cite{thygesen2012,arbab2012}. Irrespective of underlying cause, ischemia and necrosis are the common pathological characteristics of all AMI. Thrombotic MI (MI results from spontaneous atherosclerotic plaque disruption with the formation of an occluding coronary thrombus) versus non-thrombotic MI represents an important etiological distinction \cite{defilippis2017}, as both types necessitate different treatment approaches \cite{thygesen2012}. A diagnosis of AMI can be substantiated by blood-based diagnostic tests that measure isoforms of the protein troponin which is released into the circulation following necrosis \cite{newby2012}. To date a blood-based diagnostic test capable of discriminating between thrombotic and non-thrombotic MI has not been developed, although it has been previously shown that a metabolic signature may differentiate between the types \cite{defilippis2017,trainor2018}. In the present work, we set out to develop a Bayesian model for differentiating thrombotic MI from both non-thrombotic MI and stable coronary artery disease (CAD) using metabolites detected in blood plasma. We regard plasma as a promising media for developing a non-invasive test as plasma contains hormones, enzymes, lipoproteins, and other metabolic intermediates found in circulation. As metabolite concentrations are a product of genetic factors, environmental exposures, and the interaction between the two, sampling metabolites may provide a more robust characterization of the state of an organism than other approaches such as genomics. 

\section{Clinical cohort and samples}
Towards the effort of developing a Bayesian diagnostic model, we utilized previously collected samples from a patient cohort that was recruited specifically for contrasting thrombotic MI from multiple control phenotypes \cite{defilippis2015,defilippis2017}. This cohort was comprised of three phenotypic groups of human subjects: thrombotic MI, non-thrombotic MI, and stable CAD. In reference to the  thrombotic MI group, both control groups (non-thrombotic MI and stable CAD) served as procedural controls as all groups underwent a cardiac catheterization procedure. The stable CAD group provides a stable disease control as both thrombotic MI subjects and stable CAD subjects have underlying coronary artery disease. Non-thrombotic MI subjects presented with myocardial necrosis and thus the non-thrombotic MI group serves as an acute disease event control. 

Whole blood was collected shortly before cardiac catheterization from all study subjects. Details of the sample handling, sample processing, separation by liquid or gas chromatography, and mass spectrometry analysis for quantifying metabolites are provided in Chapter~\ref{hdInteractome} Section~\ref{plasma}. 

\section{Feature selection}
A significant analytical challenge in developing a blood-based diagnostic test for differentiating MI types is to determine a small set of metabolites that should be included in the statistical model from a limited number of training samples. Specifically, 1,032 chemical features (identified compounds or unknown compounds) were detected from 11 thrombotic MI, 12 non-thrombotic MI, and 15 stable CAD subjects. To ensure that the statistical model is estimable from a small sample size, and as building a targeted MRM assay or multiplexed ELISA assay can only accommodate a small number of compounds, feature selection is a critical task. While other dimension reduction techniques such as latent variable approaches (e.g. Partial Least Squares models) that create new variables which are linear combinations of metabolites would be amenable for reducing the number of coefficients to be estimated in a classification model, these approaches would not reduce the number of metabolites needing to be quantified by future targeted assays. Consequently, we prioritized reducing the number of metabolites considered. To determine a statistical classifier with five metabolites, $9.7 \times 10^{12}$ combinations are possible. In order to search the space of possible models we employed a feature selection technique that utilizes an evolutionary algorithm and seeks a consensus solution over bootstrapped datasets as described in Trainor et al. \cite{trainor2018}. In this work, small sets of metabolites were included in a multinomial logit classifier. Each model represented an individual in a population. Genetic fitness was determined as the likelihood of each individual model. These populations of models were allowed to evolve given evolutionarily inspired processes such as birth, recombination, and death. Populations were grown over bootstrapped datasets to increase diversity and reduce the correlation between models in the populations. Finally, the frequency that an individual metabolite was included in models within the final population after epochs of evolution was determined yielding a variable importance score for each metabolite. A correlation plot illustrating the metabolites with greatest variable importance score using the described technique is shown in Figure~\ref{fig:include}.

\end{DoubleSpace*}

\begin{figure}[H]
	\resizebox{1.1\textwidth}{!}{\includegraphics*{../../AthroMetab/WoAC/corrplot333}}
	\caption[Feature selection]{Metabolites with the highest variable importance score given the feature selection method we employed in a previous work \cite{trainor2018}. The Pearson correlation coefficients between metabolite transformed and scaled abundances are shown. \label{fig:include}}
\end{figure}

\begin{DoubleSpace*}
\section{Model}
A multinomial logit model was assumed for determining the  probability a sample from a clinical subject was member of the thrombotic MI, non-thrombotic MI, or a stable CAD study group. The multinomial logit model is a generalized linear model and has the following link function and model form: 
\begin{align}
\eta_{ij} = \log \frac{\pi_{ij}}{\pi_{iJ}} = \alpha_j + \textbf{x}_i^T \boldsymbol{\beta}_j,
\end{align} 
where $i$ indexes individual samples, $j$ is the index of study groups, $\boldsymbol{\beta}_j$ is a vector of regression coefficients (with separate coefficients for each group), $\alpha_j$ is a group specific intercept term, and $J$ represents the reference group. Given a specific value for the link function, the probability a sample belongs to specific study group can be computed as:
\begin{align}
\hat{\pi}_{ij} = \frac{\exp \hat{\eta}_{ij}}{\sum_{k=1}^{J}\exp \hat{\eta}_{ik}}.
\end{align}

The model can be stated as a Bayesian model with the following priors and deterministic component:
\begin{align}
\begin{split}
	\alpha_j \sim N(0,4) \\
	\beta_j \sim N(0,1) \\
	\log \frac{\pi_{ij}}{\pi_{iJ}} = \alpha_j + \textbf{x}_i^T \boldsymbol{\beta}_j \\
	Y \sim Multinom(\boldsymbol{\pi})
\end{split}
\end{align}

\section{MCMC sampling of the posterior distribution}
A MCMC sampler known as the ``No-U-Turn'' sampler was utilized to simulate the posterior distribution of model parameters \cite{hoffman2014}. This algorithm is an extension of the Hamiltonian Monte Carlo algorithm. The Hamiltonian Monte Carlo algorithm is designed to model a target probability distribution as a Hamiltonian system \cite{betancourt2017}. In such a system, a parameter vector $\boldsymbol{\theta}$ is viewed as a particle in a $D$-dimensional space \cite{neal2011} by defining a vector field that is aligned with the typical set (the region of a parameter space with both significant volume and desnity) \cite{betancourt2017}. Defining such a vector field is a complex task. While a vector field could be defined using the gradient of the target probability distribution, this vector field would pull a particle towards the mode of the distribution. By adding a momentum term, the vector field can be defined so as to restrict a particle to maintaining the Hamiltonian at a fixed value. The Hamiltonian Monte Carlo algorithm utilizes Metropolis-Hastings proposals for updating both the momentum and position variables \cite{neal2011,betancourt2017}. A critical aspect of Hamiltonian Monte Carlo sampling is determining the optimal integration time for a particle to travel along a Hamiltonian path. One approach to dynamically determining this parameter is the ``No-U-Turn'' termination criteria \cite{hoffman2014,betancourt2017}. Conceptually, this criteria ensures that expansion of a trajectory continues to visit previously unexplored neighborhoods while terminating the trajectory when it returns to previously explored neighborhoods.

The ``No-U-Turn'' Hamiltonian MCMC sampler was implemented by others in \emph{Stan}, a probabilistic programming language \cite{carpenter2017}. Stan is written in C++ and provides the ``No-U-Turn'' sampler, a Hamiltonian Monte Carlo Sampler, a variational inference algorithm, and multiple optimizers. Stan provides grammar and syntax for succinctly specifying Bayesian models; can design a sampler for the posterior distribution of model parameters; and using a C++ compiler, Stan compiles the sampler to byte code. Additionally, Stan provides data structures for storing MCMC chains and model output. An R to Stan interface, \emph{rstan} \cite{stan2018}, has been developed allowing an end user to pass datasets between Stan and to return MCMC chains and model results. 

\section{Results}
In order to simulate the posterior distribution of model parameters, four MCMC chains were generated utilizing the No-U-Turn algorithm. An example MCMC chain is illustrated in Figure~\ref{fig:brm1Coef}(a). In this figure 2,000 of the 10,000 iterations of the regression coefficient parameter for 3-hydroxypyridine sulfate from one of the MCMC chains (chain \#1) is shown. Additionally, this figure shows how the posterior distribution is generated from the MCMC chain. In Figure~\ref{fig:brm1Coef}(b), by-group parameter histograms are shown for the multinomial logit model parameters corresponding to  3-hydroxypyridine sulfate. For this metabolite, the probability mass of posterior coefficient estimates is centered slightly below zero for non-thrombotic MI, while the center of the posterior distribution is above zero for thrombotic MI. From the MCMC simulated posterior distribution, Bayesian credible intervals were determined for each model parameter as shown in Figure~\ref{fig:brm1Coef}. 

As the objective of the present analysis was to develop a model capable of discriminating between the three phenotypes utilizing only the information available at the presentation of patients and in a non-invasive manner, we also considered introducing clinical troponin values. Bayesian credible intervals are presented from the posterior distribution given the introduction of troponin in Figure ~\ref{fig:brm2Coef}. Substantial qualitative changes in the credible intervals were not observed following the introduction of troponin. To compare the models, the ``Widely applicable information criterion'' was utilized (see Table~\ref{tab:modelComp}). A $\chi^2$ test on one degree of freedom reveals that troponin did not lead to a significant improvement overall. 

\begin{table}[H]
	\centering
	 	\caption{text}
	 	\label{tab:modelComp}
	\begin{tabular}{lcc}
		\hline
       Model & WAIC  & SE \\
       \hline
       Without troponin & 15.43 & 4.05 \\ 
       With troponin & 14.60 & 3.72 \\
       Difference & 0.83 & 0.87 \\
       \hline
	\end{tabular}
\end{table}

\end{DoubleSpace*}

\newpage
\KOMAoptions{paper=landscape}
\recalctypearea
\begin{figure}[H]
	\includegraphics[scale=1.1]{../Aim3/MCMCEx.png}
	\caption[Add caption]{Add caption \label{fig:brm1Coef} }
\end{figure}
\newpage
\begin{figure}[H]
	\includegraphics[scale=1.15]{../Aim3/brm1Coef}
	\caption[Add caption]{Add caption \label{fig:brm1Coef} }
\end{figure}
\newpage
\begin{figure}[H]
	\includegraphics[scale=1.15]{../Aim3/brm2Coef}
	\caption[Add caption]{Add caption \label{fig:brm2Coef} }
\end{figure}
\newpage
\KOMAoptions{paper=portrait,pagesize}
\recalctypearea

\begin{DoubleSpace*}
Given that the MCMC chains provide the joint posterior distribution of model parameters, the correlation between parameter estimates can be evaluated. Two parameter estimates were significantly correlated, both of which were for monoacylglycerols [1-linoleoylglycerol (18:2)] and [2-linoleoylglycerol (18:2)]. This relationship was further explored. Figure~\ref{fig:coefPost} shows a 3-dimensional scatterplot of samples from the simulated posterior distribution. In the first plane, the parameter estimates for both monoacylglycerols are shown, while the remaining axis shows the log-posterior probability of the model with these parameterizations.  Similarly, Figure~\ref{fig:coefPost2} shows the same slice of the posterior distribution with the log-posterior probability axis removed and represented as a color instead. The relatively strong negative correlation between the parameter estimates, without a systemic change in the log-posterior probability suggests that these two monoacylglycerols modulate the probability of the phenotypes in a similar way. 
\end{DoubleSpace*}
\begin{figure}[H]
	\resizebox{.8\textwidth}{!}{\includegraphics*{../Aim3/coefPost}}
	\caption[Add caption]{Add caption \label{fig:coefPost} }
\end{figure}

\begin{figure}[H]
	\resizebox{1.1\textwidth}{!}{\includegraphics*{../Aim3/coefPost2}}
	\caption[Add caption]{Add caption \label{fig:coefPost2} }
\end{figure}

From the simulated posterior distribution of models, a predictive phenotype distribution can be generated for each human subject. The MCMC chain for the phenotype probabilities of a selected clinical subject is presented in Figure~\ref{fig:ptid2010MCMC}.

\begin{figure}[H]
	\resizebox{1.15\textwidth}{!}{\includegraphics*{../Aim3/ptid2010MCMC}}
	\caption[Add caption]{Add caption \label{fig:ptid2010MCMC} }
\end{figure}

\begin{figure}[H]
	\resizebox{1.15\textwidth}{!}{\includegraphics*{../Aim3/ptid2010Hist}}
	\caption[Add caption]{Add caption \label{fig:ptid2010Hist} }
\end{figure}
 
Leave-one-out cross validation estimates of study group membership for each clinical sample. Group membership was determined by maximum a posteriori estimates, and the class with maximum estimated probability of membership is reported.
 
 \begin{table}[H]
	\centering
 	\caption{text}
 	\label{tab:modelRes}
 	\begin{tabular}{l|ccc}
 		& Predicted & & \\
 		Group  &     sCAD & Thrombotic MI & Non-Thrombotic MI \\
 		\hline
 		sCAD   &     13    &     0    &     2\\
 		Thrombotic MI  &  0    &    10    &     1\\
 		Non-Thrombotic MI &   1    &     1   &     10
 	\end{tabular}
 \end{table}
 
  \begin{table}[H]
  	\centering
  	\caption{text}
  	\label{tab:modelRes2}
  	\begin{tabular}{l|ccc}
  		& Predicted & & \\
  		Group  &     sCAD & Thrombotic MI & Non-Thrombotic MI \\
  		\hline
  		sCAD   &     13    &     0    &     2\\
  		Thrombotic MI  &  0    &    10    &     1\\
  		Non-Thrombotic MI &   2  &   0 &    10
  	\end{tabular}
  \end{table}
  
  \begin{figure}[H]
	  	\centering
  	  	\caption{text}
 	  	\label{tab:modelRes3}
  	\includegraphics[scale=.85]{../Aim3/PatientPred}
  \end{figure}