\documentclass[xcolor=dvipsnames]{beamer} 
\usetheme{AnnArbor}
\usecolortheme{beaver}

\usepackage{amsmath,graphicx,booktabs,tikz,subfig,color}
\definecolor{mycol}{rgb}{.4,.85,1}
\setbeamercolor{title}{bg=mycol,fg=black} 
\setbeamercolor{palette primary}{use=structure,fg=white,bg=red}
\setbeamercolor{block title}{fg=white,bg=red!50!black}
%\setbeamercolor{block title}{fg=white,bg=blue!75!black}

\DeclareMathOperator{\PP}{P}
\DeclareMathOperator{\EE}{E}
\DeclareMathOperator{\argmin}{argmin}
\DeclareMathOperator{\argmax}{argmax}
\DeclareMathOperator{\ent}{H}
\DeclareMathOperator{\tr}{tr}
\DeclareMathOperator{\sign}{sign}
\DeclareMathOperator{\DE}{DE}
\DeclareMathOperator{\EXP}{EXP}
\DeclareMathOperator{\GA}{GA}
\DeclareMathAlphabet\mathbfcal{OMS}{cmsy}{b}{n}

\begin{document}
	
\title[Bayesian Methods for Metabolomics]{{\bf Bayesian analytical approaches for metabolomics:}\\ {\small A novel method for molecular structure-informed probabilistic metabolite interaction modeling, a novel diagnostic model for differentiating Myocardial Infarction type, and approaches for compound identification}}
\author[P.J. Trainor]{Patrick J. Trainor}
\institute[U of L]
{
	% \includegraphics[scale=.4]{/home/patrick/gdrive//MetabClass/Old/RL2015/IMClogo}\\
	Interdisciplinary Ph.D. Program in Bioinformatics \& \\
	Division of Cardiovascular Medicine  \\
	University of Louisville %\\[2ex]
}
\date[July 2018]{July 19, 2018}

\begin{frame}
	\titlepage
\end{frame}

\begin{frame}{Introduction}{Probabilistic interaction modeling (Interactomes)}
	\vspace{-15pt}
	\begin{center}
		\includegraphics[scale=.64]{path2}
	\end{center}
\end{frame}

\begin{frame}{Introduction}{Generating a diagnostic model for Acute MI}
	\vspace{-15pt}
	\begin{center}
		\includegraphics[scale=.9]{xcms2Class}
	\end{center}
\end{frame}

\begin{frame}{Introduction}{Methods for compound identification}
	\vspace{-15pt}
	\begin{center}
		\includegraphics[scale=.8]{compID}
	\end{center}
\end{frame}

\begin{frame}{}{}
	\vspace{-15pt}
	{\Huge
	The thread that binds the aims: \pause
	\begin{align*}
		&\downarrow n \; \text{(samples)} \\
		&\uparrow p  \; \text{(metabolites)}
	\end{align*}} \pause 
	{\LARGE \\the integration of extra-experimental scientific knowledge is vital}

\end{frame}

\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[hideallsubsections]
\end{frame}

\section{Probabilistic interaction modeling (Interactome)}
\subsection{Motivation: Higher order inference in metabolomics}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=hide]
\end{frame}

\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Motivation}
	\vspace{-15.5pt}
	{\LARGE In someone else's words...}
	\begin{center}
		\includegraphics[scale=.5]{../Metabolomics2018/JohnsonNatureReview}
	\end{center}
	(2016). \emph{Nature Rev Mol Cell Bio, 17}. doi: 10.1038/nrm.2016.25
\end{frame}

\begin{frame}{Motivation}
	\vspace{-10pt}
		{\LARGE A tale of two aims:}
		
		\begin{enumerate}[{1)}]
			\item ``...Aim 2 determines which TAM metabolic pathway(s) is regulated by c-Maf using systems metabolomics approach. We will first determine TAM metabolic pathways using Stable Isotope Resolved Metabolomics (SIRM) approach...'' [1R01CA213990-01] \pause
			\item[]
			\item ``...The objective of the proposed project is to perform untargeted metabolite profiling to reveal metabolite abnormalities that are common to sALS patients and to determine metabolic pathways whose dysregulation contributes to the disease...'' [5F31NS095698-02]
		\end{enumerate}
\end{frame}

\begin{frame}{Motivation}{Biomarker hypotheses}
	\vspace{-10pt}
	You are interested in whether a therapeutic ``changes'' the abundance of a metabolite, $X$ in plasma (e.g. a specific lipid) \pause
	\begin{itemize}
		\item $H_0$: The abundance of $X$ does not differ between two (or more) phenotypes \pause
		\item $H_A$: The abundance of $X$ does differ between two (or more) phenotypes 
		\item[]
	\end{itemize} \pause
	
	Or formulated as an equivalence test:
	\begin{itemize}
		\item $H_0$: The abundance of $X$ is not equivalent between two (or more) phenotypes  
		\item $H_A$: The abundance of $X$ is equivalent between two (or more) phenotypes  
	\end{itemize}
\end{frame}

\begin{frame}
	\begin{center}
			\includegraphics[scale=.4]{../Plots/cholesterol.png}
			
			Lieberman, M., Ahles, P., et al. \emph{Wikipathways}: WP197
	\end{center}
\end{frame}

\begin{frame}{Motivation}{Hypotheses}
	\vspace{-10pt}
	You are interested in whether a therapeutic ``changes'' cholesterol biosynthesis \pause
	\begin{itemize}
		\item $H_0$: cholesterol biosynthesis does not differ between two (or more) phenotypes 
		\item $H_A$: cholesterol biosynthesis does differ between two (or more) phenotypes \pause
		\item[]
	\end{itemize} 
	
	Or formulated as an equivalence test:
	\begin{itemize}
		\item $H_0$: cholesterol biosynthesis is the same between two (or more) phenotypes 
		\item $H_A$: cholesterol biosynthesis is not the same between two (or more) phenotypes
	\end{itemize}
\end{frame}

\begin{frame}
	\begin{center}
		\includegraphics[scale=.4]{../Plots/cholesterol.png}
		
		Lieberman, M., Ahles, P., et al. \emph{Wikipathways}: WP197
	\end{center}
\end{frame}

\begin{frame}{Motivation}{Higher order inference}
	\vspace{-10pt}
	{\Large Higher-order inference: the testing of hypotheses that are more complex than single metabolite hypothesis tests} \vspace{7pt} \pause
	
	\begin{itemize}
		\item \textbf{``Data dependent'' / ``Unbiased'' Approaches:} 
		\begin{itemize}
		\item Analyses of correlation networks
		\item Module analyses (WGCNA)
		\item Multivariate statistical approaches (Clustering; Latent component modeling such as PCA, PLS-R, PLS-DA) \pause
		\end{itemize}
		\item[]
		\item \textbf{\emph{A priori approaches}--Pathway (or other ontology) enrichment analyses:} 
		\begin{itemize}
			\item Discrete statistical tests (e.g. hypergeometric)
			\item KS statistic-based set enrichments (e.g. GSEA/MSEA)
		\end{itemize}
	\end{itemize}
	
\end{frame}

\begin{frame}{Pathway enrichment analysis}
	\vspace{-10pt}
{\Large Test for pathway enrichment: Does a pathway have more ``significant'' differences in metabolite abundances than expected?} \vspace{10pt} \pause

	\begin{itemize}
		\item pmf for the hypergeometric distribution: 
		\begin{itemize}
			\item $N=$ total metabolites,
			\item $K=$ number of differentially abundant metabolites, 
			\item $n=$ metabolites in the pathway of interest, 
			\item $k=$ number of differentially abundant metabolites in the pathway of interest 
		\end{itemize}
		\begin{align*}
		\PP(X=k) = \frac{  {{K}\choose{k}} {{N-K}\choose{n-k}}  }{ {{N}\choose{n}} } 
		\end{align*}
	\end{itemize}
\end{frame}

\begin{frame}{Bias in pathway enrichment analysis}{The reference set}
	\vspace{-15.5pt}
	\begin{figure}
		\includegraphics[scale=.32]{/home/patrick/gdrive/Dissertation/Proposal/vennHell2_1}
	\end{figure}
\end{frame}

\begin{frame}{Bias in pathway enrichment analysis}{The reference set}
	\vspace{-15.5pt}
	\begin{figure}
		\includegraphics[scale=.32]{/home/patrick/gdrive/Dissertation/Proposal/vennHell2_2}
	\end{figure}
\end{frame}

\begin{frame}{Bias in pathway enrichment analysis}{Metabolic footprint vs. cellular metabolism}	\vspace{-10.5pt}
	\begin{center}
		\includegraphics[scale=.3]{../Metabolomics2018/512px-Circulatory_System_en}
	\end{center}
\end{frame}

\begin{frame}{Correlation networks}{What do they tell you?}
	{I simulated this...}
	\begin{center}
		\includegraphics[scale=.5]{../../Aim2/Plots/AR1_Om}
	\end{center}
\end{frame}

\begin{frame}{Correlation networks}{What do they tell you?}
	{And fit a correlation network...}
	\begin{center}
		\includegraphics[scale=.5]{../../Aim2/Plots/AR1_Cor}
	\end{center}
\end{frame}

\begin{frame}{Partial correlation}{A better approach}
	\vspace{-15.5pt}
	{\Large \textbf{Gaussian Graphical Models (GGM)}: from correlation to partial correlation...}
	
	\begin{center}
		\includegraphics[scale=.75]{../Metabolomics2018/ggmPrev} \\
		Krumsiek et al. (2011). \emph{BMC Syst Bio, 5}(21) doi: 10.1186/1752-0509-5-21
	\end{center}
\end{frame}

\begin{frame}{Partial correlation}{A better approach}
	\vspace{-15.5pt}
	\begin{center}
		\includegraphics[scale=.75]{../Metabolomics2018/ggmPrev} \\
		Krumsiek et al. (2011). \emph{BMC Syst Bio, 5}(21) doi: 10.1186/1752-0509-5-21
	\end{center} \pause
	\begin{itemize}
		\item Problem: requires estimating $\frac{p*(p-1)}{2}$ coefficients
		\item[]
		\item Not feasible for most metabolomics studies 
	\end{itemize}
\end{frame}

\begin{frame}{Motivation}
	\vspace{-15.5pt}
	\begin{itemize}
		\item Succinct problem statement: \pause
		\begin{itemize}
			\item To make higher-order inference a model of how metabolites are related is required \pause
			\item[]
			\item Unbiased approach (using a GGM to estimate partial correlation coefficients) is often unfeasible \pause
			\item[]
			\item Pathway-based approach may be inappropriate (given the experimental system, sample media, etc.) \pause
		\end{itemize}
		\item[]
		\item A Bayesian call to action!
	\end{itemize}
\end{frame}

\begin{frame}{Bayesian call to action}
	
	{\LARGE Frameworks}
	\begin{itemize}
		\item Today: \pause
		\begin{enumerate}[{1)}]
			\item Collect experimental data \pause
			\item Do frequentist analysis \pause
			\item Use your expert knowledge of metabolism to interpret what the results mean ``I knocked out DHCR7 which explains why there are changes in 7-Dehydrocholesterol and Cholesterol'' \pause
		\end{enumerate}
		\item[]
		\item Hopefully the future:
		\begin{enumerate}[{1)}]
			\item Use knowledge of metabolism to formulate informative (or weakly informative) priors \pause
			\item Collect experimental data \pause
			\item Evaluate posterior distributions to make inference
		\end{enumerate}
	\end{itemize}
\end{frame}

\subsection{Bayesian reasoning}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Bayesian reasoning example}
\vspace{-15pt}
\begin{itemize}
	\item Mathematical foundation:
	\begin{align*}
	p(\theta|\textbf{x})&=\frac{p(\textbf{x}|\theta)p(\theta)}{\int_{\tilde{\theta} \in \Theta} p(\textbf{x}|\tilde{\theta})p(\tilde{\theta})d\tilde{\theta}}\\ &=\frac{p(\textbf{x}|\theta)p(\theta)}{p(\textbf{x})}\\
	&\propto p(\textbf{x}|\theta)p(\theta).
	\end{align*}
	\item Example:
	\begin{itemize}
		\item Mass spectrometer with a counting detector that records the counts per second over a fixed $m/z$ window
		\item $X\sim Pois(\lambda)$
		\item Sample: $\{107, 103, 110, 99, 108, 108, 105\}$
	\end{itemize}
\end{itemize}
\end{frame}

\begin{frame}{Bayesian reasoning example}
	\vspace{-15pt}
	\begin{itemize}
		\item Poisson pmf:
				\begin{gather*}
				f(x|\lambda)=\frac{\lambda^x e^{-\lambda}}{x!}, \quad x \in \mathbb{N}, \quad \lambda \in \left[0, \infty \right)
				\end{gather*}
		\item Ten prior count observations and that the sum of counts from these observations was 1,000
		\item Prior distribution for $\lambda$ of $Gamma(\alpha, \beta)$ with $\alpha=1000$, $\beta=10$
		\item Gamma pdf:
		\begin{gather*}
		f(\lambda|\alpha,\beta) =\frac{\beta^{\alpha}}{\Gamma(\alpha)}\lambda^{\alpha-1}e^{-\beta \lambda}, \quad \lambda \in \left[0, \infty \right), \quad \alpha,\beta >0.
		\end{gather*}
		\item \textbf{Goal}:  $p(\textbf{x}|\lambda)p(\lambda)$
	\end{itemize}
\end{frame}

\begin{frame}{Bayesian reasoning example}
		We note that the likelihood for the Poisson distribution factorizes as in the following:
		\begin{align*}
		\mathcal{L}(\lambda|\textbf{x})=p(\textbf{x}|\lambda)=\prod_{i=1}^n \frac{\lambda^{x_i}e^{-\lambda}}{x_i!}= \left( \prod_{i=1}^n x_i! \right)^{-1} \times \lambda^{\sum_{i=1}^n x_i} e^{-n\lambda}. 
		\label{eq:poissonLik}
		\end{align*}
		Considering the prior distribution that we have specified:
		\begin{align*}
		p(\lambda|\textbf{x})&\propto \frac{\beta^{\alpha}}{\Gamma(\alpha)}\lambda^{\alpha-1}e^{-\beta \lambda} \lambda^{\sum_{i=1}^n x_i} e^{-n\lambda} \\
		&\propto \frac{\beta^{\alpha}}{\Gamma(\alpha)} \lambda^{\alpha + \sum_{i=1}^n x_i -1} e^{-(\beta+n)\lambda},
		\end{align*}
		we can note that:
		\begin{align*}
		p(\lambda|\textbf{x}) =c(\textbf{x},\alpha,\beta)^{-1} \lambda^{\alpha + \sum_{i=1}^n x_i -1} e^{-(\beta+n)\lambda}
		\end{align*}
\end{frame}

\begin{frame}{Bayesian reasoning example}
	\vspace{-5pt}
	\begin{center}
		$p(\lambda|\textbf{x})\sim Gamma(\alpha+ \sum_{i=1}^n x_i, \beta+n)$
		\includegraphics[scale=.6]{../Plots/Poisson}
		
		$\hat{\lambda}_{MLE}=105.7143, \quad \tilde{\lambda}=102.2941$
	\end{center}
\end{frame}

\subsection{The current model}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Gaussian Graphical Models (GGMs)}
	\vspace{-5.5pt}
	\begin{itemize}
		\item Markov Random Fields: A graph $G=(V,E)$ in which random variables $X_i\in V$, $i\in \{1,2,...p\}$ are represented by vertices and edges in the edge set $E \subseteq V \times V$ represent probabilistic interactions \pause
		\item[]
		\item GGM: $\textbf{X}\sim \mathcal{N}(\boldsymbol{\mu},\boldsymbol{\Omega}^{-1})$ where $\boldsymbol{\Omega}$ is the concentration matrix (inverse of covariance matrix) \pause
		\item[]
		\item Gaussian conditional distributions and marginal distributions \pause
		\item[]
		\item From a GGM you can determine partial correlation coefficients 
	\end{itemize}
\end{frame}

\begin{frame}{Gaussian Graphical Models (GGMs)}
	\vspace{-15.5pt}
	\begin{itemize}
		\item Likelihood:
		\begin{align*}
		L(\boldsymbol{\Omega}|\textbf{X})&=(2 \pi)^{-np/2}|\boldsymbol{\Omega}|^{n/2} \exp \left(-\frac{1}{2}\sum_{i=1}^{n} (\textbf{x}_i-\boldsymbol{\mu})^T \boldsymbol{\Omega} (\textbf{x}_i-\boldsymbol{\mu})\right) 
		\end{align*}\pause
		\item Proportionally:
		\begin{align*} 
		l(\boldsymbol{\Omega}|\textbf{S})\propto\log (\det \boldsymbol{\Omega})-\tr \left( \textbf{S} \boldsymbol{\Omega} \right)
		\end{align*}\pause
		\item[]
		\item Given the experimental design of many/most metabolomics studies the likelihood would be non-convex
	\end{itemize}
\end{frame}

\begin{frame}{GGM estimation}
	\vspace{-15.5pt}
	\begin{itemize}
		\item $L_1$ regularized likelihood:
		\begin{align*}
		l(\boldsymbol{\Omega})\propto\log (\det \boldsymbol{\Omega})-\tr \left( \textbf{S} \boldsymbol{\Omega} \right)-\lambda ||\boldsymbol{\Omega}||_1
		\end{align*}\pause
		\item Method for optimizing: the Graphical Lasso. Friedman, J., et al. (2007). \emph{Biostatistics, 9}. doi: 10.1093/biostatistics/kxm045 \pause
		\item[]
		\item Adaptive graphical Lasso: Zou, H. (2006).  \emph{J Amer Stat Assoc, 101}. doi: 10.1198/016214506000000735 
		\begin{align*}
				l(\boldsymbol{\Omega})\propto \log(\det \boldsymbol{\Omega})-\tr \left(\mathbf{S} \boldsymbol{\Omega}\right) - \lambda \sum_{1\leq i \leq p} \sum_{1 \leq j \leq p} w_{ij} |\omega_{ij}|.
		\end{align*}
		with adaptive weights: $w_{ij}=|\hat{\omega}_{ij}|^\alpha$
	\end{itemize}
\end{frame}

\begin{frame}{Bayesian estimation}
	\vspace{-5.5pt}
\begin{itemize}
	\item A hierarchical Bayesian model for the graphical Lasso (Wang, H. [2012]. \emph{Bayesian Analysis, 7}. doi: 10.1214/12-ba729): 
	
		\begin{align*}
		p(\textbf{x}_i|\boldsymbol{\Omega}) =& \mathcal{N}(\textbf{0},\boldsymbol{\Omega}^{-1}) \quad \text{for} \; i=1,2,\hdots,n\\
		p(\boldsymbol{\Omega}|\lambda) =& \frac{1}{C} \prod_{i<j} \DE(\omega_{ij}|\lambda) \prod_{i=1}^{p} \EXP (\omega_{ii} | \lambda / 2) \cdot 1_{\boldsymbol{\Omega}\in M^+},
		\end{align*} \pause
	\item[]
	\item As a scale mixture of Gaussian distributions:
\end{itemize}
		\begin{align*}
		p(\boldsymbol{\omega}| \boldsymbol{\tau},\lambda)=\frac{1}{C_{\boldsymbol{\tau}}} \prod_{i<j} \left[ \frac{1}{\sqrt{2\pi \tau_{ij}}} \exp \left(- \frac{\omega_{ij}^2}{2\tau_{ij}}\right) \right] 
		\prod_{i=1}^{p}  \left[\frac{\lambda}{2} \exp \left(-\frac{\lambda}{2}\omega_{ii} \right)\right] \cdot 1_{\boldsymbol{\Omega}\in M^+}
		\end{align*}
\end{frame}

\begin{frame}{Bayesian estimation}
	\vspace{-15.5pt}
	Also a hierarchical Bayesian model for the adaptive graphical Lasso (Wang, H. [2012]. \emph{Bayesian Analysis, 7}. doi: 10.1214/12-ba729):
	
	\begin{align*}
	p(\mathbf{x}_i|\boldsymbol{\Omega}) = & \mathcal{N}(\mathbf{0,\boldsymbol{\Omega}}^{-1}) \quad \text{for} \; i=1,2,\hdots,n\\
	p(\boldsymbol{\Omega}|\{\lambda_{ij}\}_{i\leq j}) = & C^{-1} \prod_{i<j} \DE(\omega_{ij}|\lambda_{ij}) \prod_{i=1}^{p} \EXP (\omega_{ii} | \lambda_{ii} / 2) \cdot 1_{\boldsymbol{\Omega}\in M^+}\\
	p(\{\lambda_{ij}\}_{i<j}|\{\lambda_{ii}\}_{i=1}^p) &\propto C_{\{\lambda_{ij}\}_{i\leq j}} \prod_{i<j} \GA(r,s)
	\end{align*}
\end{frame}

\subsection{Using molecular similarity for estimation}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}
\begin{frame}{An informative Bayesian estimation}{The current work}
	\vspace{-15.5pt}
	\begin{itemize}
		\item \textbf{Idea}:  incorporate prior knowledge regarding the substructures that are shared between compounds (molecular similarity). That is:
		\begin{align*}
			\lambda_{ij}\sim Gamma(r,s_{ij})
		\end{align*}\pause
		\item[]
		\item[]
		\item Conditional expectation: $\EE(\lambda_{ij}|\boldsymbol{\Omega})=(1+r)/(|\omega_{ij} |+s_{ij})$.
	\end{itemize}
\end{frame}

\begin{frame}{Informative Bayesian estimation}{The current work}
	\begin{center}
			\includegraphics[scale=.7]{../../Aim2/Plots/lambdasVsSim}
	\end{center}
\end{frame}

\begin{frame}{Informative Bayesian estimation}{Block Gibbs Sampler}
	\vspace{-10.5pt}
	\begin{itemize}
		\item Need a way to simulate the posterior distribution \pause
		\item[]
		\item Gibb's sampling is a MCMC technique for simulating a target distribution if conditional distributions are known \pause
		\item[]
		\item Conditional distribution for concentration matrix columns:
	\end{itemize}
	\begin{align*}
	p(\boldsymbol{\omega}_{12}, \omega_{22}|\boldsymbol{\Omega}_{11},\mathbfcal{T},\textbf{X},\lambda) & \propto \left(\omega_{22}-\boldsymbol{\omega}_{12}^T \boldsymbol{\Omega}_{11}^{-1}\boldsymbol{\omega}_{12} \right)^{n/2} \\ &\exp \left( - \frac{1}{2}\left[ \boldsymbol{\omega}_{12}^T \textbf{D}_{\boldsymbol{\tau}} \boldsymbol{\omega}_{12}+ 2 
	\textbf{s}_{12}^T \boldsymbol{\omega}_{12} + (s_{22}+\lambda)\omega_{22}\right] \right)
	\end{align*}
\end{frame}

\subsection{Computational nuances}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}
\begin{frame}{Informative Bayesian estimation}{Block Gibbs Sampler}
	\vspace{-15.5pt}
	\begin{itemize}
		\item Developed an R package for implementing this sampler: \emph{BayesianGLasso} \pause
		\item[]
		\item Sampler is written in C++ utilizing the Armadillo linear algebra library \pause
		\item[]
		\item To interface between R and C++, utilized \emph{Rcpp} and \emph{RcppArmadillo} \pause
		\item[]
		\item To utilize with $\approx 500$ metabolites significant optimization was required (openBLAS \& LAPACK routines)
	\end{itemize}
\end{frame}

\begin{frame}{Simulated experiment}
	\begin{center}
		\includegraphics[scale=.33]{../../Aim2/Plots/AR1}
	\end{center}
\end{frame}

\subsection{A Heart Disease interactome}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}
\begin{frame}{Heart Disease Interactome}
	\vspace{-15.5pt}
	\begin{itemize}
		\item Goal: We sought to construct a stable heart disease plasma metabolite interactome \pause
		\item[]
		\item Long term motivation:  to serve as a reference for analyzing metabolic changes to an acute state (MI) \pause
		\item[]
		\item 47 plasma samples from human subjects with heart disease \pause
		\item[]
		\item 522 compounds identified from and quantified by UPLC-MS/MS and GC-MS
	\end{itemize}
\end{frame}

\begin{frame}{Heart Disease Interactome}
		\vspace{-15.5pt}
	\begin{center}
		\includegraphics[scale=.35]{../../Aim2/Plots/StructHeatmaps}
		
		Local substructure similarity measure adapted from Mitchell, J. (2014). doi: 10.3389/fgene.2014.00237 (collaboration)
	\end{center}
\end{frame}

\begin{frame}{Heart Disease Interactome}{MCMC sampling}
	\vspace{-10.5pt}
	\begin{center}
		\includegraphics[scale=.52]{../../Aim2/Plots/MCMC}
	\end{center}
\end{frame}

\begin{frame}{Heart Disease Interactome}
	\vspace{-10.5pt}
	\begin{center}
		\includegraphics[scale=.52]{../../Aim2/Plots/aiBGL1CorBigGraph}
	\end{center}
\end{frame}

\begin{frame}{Heart Disease Interactome}
	\vspace{-10.5pt}
	\begin{center}
		\includegraphics[scale=.175]{../../Aim2/Plots/aiBGL1Cor(2)}
	\end{center}
\end{frame}

\begin{frame}{The future}
	\vspace{-15pt}
	\begin{center}
		\includegraphics[scale=.225]{../Metabolomics2018/FlowChart2}
		
		Trainor, P. (2018), \emph{J Biomed Info, 81}. doi: 10.1016/j.jbi.2018.03.007 \pause
	\end{center}
	\begin{itemize}
		\item Use the stable disease interactome to determine systems-levels changes that occur in the relationships between metabolites in plasma following acute disease events (e.g. heart attacks) \pause
		\item[]
		\item Prior distributions 
	\end{itemize}
\end{frame}

\section{Diagnostic model for Acute MI}
\subsection{Acute Myocardial Infarction}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=hide]
\end{frame}

\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Acute Myocardial Infarction}
	\vspace{-5pt}
	\begin{itemize}
		\item Characterized by myocardial ischemia (exposure of cardiac myocytes to oxygen deprivation) and necrosis (a form of cell death)
	\end{itemize}
	\begin{center}
		\includegraphics[scale=.7]{miType}
		
		Thygesen, et al. (2012). doi: 10.1016/j.jacc.2012.08.001
	\end{center}
\end{frame}

\begin{frame}{Acute Myocardial Infarction}
		\vspace{-5pt}
		\begin{itemize}
			\item Criteria utilized for diagnosing MI (EKG, Troponin, symptoms) do not definitively rule in or out presence of a thrombus \pause
			\item Troponin elevation is not sufficient for determining the cause of an MI \pause
		\end{itemize}
		\begin{center}
			\includegraphics[scale=.4]{elevTrop}
			
			Newby, et al. (2012). doi: 10.1016/j.jacc.2012.08.969
		\end{center}
\end{frame}

\begin{frame}{Objective}
	\vspace{-10pt}
	\begin{itemize}
		\item Metabolite concentrations are a product of genetic factors, environmental exposures, and gene $\times$ environment \pause
		\item[]
		\item Metabolite concentrations are dynamic (especially in blood) \pause
		\item[]
		\item \textbf{Implicit hypothesis:} If we quantify metabolites in plasma will observe evidence of:
		\begin{itemize}
			\item Myocardial ischemia and necrosis (all MI types)
			\item Metabolic processes specific to thrombotic MI (platelet activation, platelet aggregation, fibrinolysis) \pause
			\item[]
		\end{itemize}
		\item \textbf{Goal:} Utilize this to construct a non-invasive diagnostic test capable of discriminating between thrombotic and non-thrombotic MI
	\end{itemize}
\end{frame}
%In the present work, we set out to develop a Bayesian model for differentiating thrombotic MI from both non-thrombotic MI and stable coronary artery disease (CAD) using metabolites detected in blood plasma. We regard plasma as a promising media for developing a non-invasive test as plasma contains hormones, enzymes, lipoproteins, and other metabolic intermediates found in circulation. As metabolite concentrations are a product of genetic factors, environmental exposures, and the interaction between the two, sampling metabolites may provide a more robust characterization of the state of an organism than other approaches such as genomics. 

\subsection{Clinical cohort \& samples}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Cohort}
	\vspace{-10pt}
	\begin{center}
		\includegraphics[scale=.65]{cohort}
		
		DeFilippis, Trainor, et al. (2017). doi: 10.1371/journal.pone.0175591
	\end{center}
\end{frame}

\subsection{Statistical model for discriminating MI type}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Statistical model}
	\vspace{-10.5pt}
	\begin{itemize}
		\item Multinomial logistic regression model:
			\begin{align*}
			\eta_{ij} = \log \frac{\pi_{ij}}{\pi_{iJ}} = \alpha_j + \textbf{x}_i^T \boldsymbol{\beta}_j,
			\end{align*} \pause
		\item Parameters:
		\begin{itemize}
			\item $i$ indexes individual samples
			\item $j$ is the index of study groups
			\item $\textbf{x}_i$ vector of metabolite abundances for the individual $i$
			\item $\boldsymbol{\beta}_j$ is a vector of regression coefficients 
			\item $\alpha_j$ is a group specific intercept term
			\item $J$ represents the reference group.
		\end{itemize} \pause
		\item Membership probability for each group:
		\begin{align*}
		\hat{\pi}_{ij} = \frac{\exp \hat{\eta}_{ij}}{\sum_{k=1}^{J}\exp \hat{\eta}_{ik}}.
		\end{align*}
	\end{itemize}
\end{frame}

\begin{frame}{Statistical model}
	\vspace{-10.5pt}
	\begin{itemize}
		\item As a Bayesian model with the following priors and deterministic component:
		\begin{align*}
		\begin{split}
		\alpha_j &\sim N(0,4) \\
		\beta_j &\sim N(0,1) \\
		\log \frac{\pi_{ij}}{\pi_{iJ}} &= \alpha_j + \textbf{x}_i^T \boldsymbol{\beta}_j \\
		Y &\sim Multinom(\boldsymbol{\pi})
		\end{split}
		\end{align*}
	\end{itemize}
\end{frame}

\begin{frame}{Feature selection}
	\vspace{-10pt}
	\begin{center}
		\includegraphics[scale=.09]{../../../AthroMetab/WoAC/corrplot333}
		
		Trainor, P. (2018). doi: 10.1016/j.jbi.2018.03.007 
	\end{center}
\end{frame}

\subsection{Computational nuances}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Hamiltonian Systems}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.375]{phase}
		
		Lecture notes Physics 3210, Dr. Ethan Neil, CU Boulder
	\end{center}
\end{frame}

\begin{frame}{Target distributions as vector fields}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.6]{HMC1}
		
	Betancourt. (2017). arXiv:1701.02434
	\end{center}
\end{frame}

\begin{frame}{Target distributions as vector fields}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.55]{HMC2}
		
		Betancourt. (2017). arXiv:1701.02434
	\end{center}
\end{frame}

\begin{frame}{The No-U-Turn sampler}
	\vspace{-10pt}
	\begin{center}
		\includegraphics[scale=.48]{nuts}
		\vspace{2ex}
		Hoffman \& Gelman. (2014). \emph{Journal of Machine Learning Research}. dl.acm.org/citation.cfm?id=2638586
	\end{center}
\end{frame}

\subsection{Model parameters}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{MCMC for a metabolite parameter}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.5]{../../Aim3/MCMCEx.png}
	\end{center}
\end{frame}


\begin{frame}{Model coefficients}{No troponin}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.575]{../../Aim3/brm1Coef}
	\end{center}
\end{frame}

\begin{frame}{Model coefficients}{With troponin}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.575]{../../Aim3/brm2Coef}
	\end{center}
\end{frame}

\begin{frame}{Model coefficients}{Posterior distribution}
		\vspace{-5pt}
		\begin{center}
			\includegraphics[scale=.15]{../../Aim3/coefPost}
		\end{center}
\end{frame}

\begin{frame}{Model coefficients}{Posterior distribution}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.6]{../../Aim3/coefPost2}
	\end{center}
\end{frame}

\subsection{Model goodness of fit \& predictions}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{To include troponin?}
	\vspace{-10.5pt}
	\begin{center}
		\begin{tabular}{lcc}
			\hline
			Model & WAIC  & SE \\
			\hline
			Without troponin & 15.43 & 4.05 \\ 
			With troponin & 14.60 & 3.72 \\
			Difference & 0.83 & 0.87 \\
			\hline
		\end{tabular}
		
		$\chi^2$-test $p=0.36$
	\end{center}
\end{frame}

\begin{frame}{Markov chain for a human subject}
	\vspace{-5pt}
	\begin{center}
		\includegraphics[scale=.6]{../../Aim3/ptid2010MCMC}
	\end{center}
\end{frame}

\begin{frame}{Markov chain for a human subject}
	\vspace{-7pt}
	\begin{center}
		\includegraphics[scale=.65]{../../Aim3/ptid2010Hist}
	\end{center}
\end{frame}

\begin{frame}{LOO-CV estimation of error}
	\vspace{-15pt}
	Model estimated by Maximum Likelihood Estimation (non-Bayesian):
	\vspace{4ex}
	
	\begin{tabular}{l|ccc}
		& \emph{Predicted} & & \\
		\emph{Actual}  & \textbf{sCAD} & \textbf{Thrombotic MI} & \textbf{Non-Thromb. MI} \\
		\hline
		\textbf{sCAD} & 13 &  0 & 2\\
		\textbf{Thrombotic MI} &   1 & 9 &  1\\
		\textbf{Non-Thromb. MI}  & 2  & 2 & 8 
	\end{tabular}
\end{frame}

\begin{frame}{LOO-CV estimation of error}
	\vspace{-15pt}
	Bayesian model without troponin:
	\vspace{4ex}

\begin{tabular}{l|ccc}
	& \emph{Predicted} & & \\
	\emph{Actual} &  \textbf{sCAD} & \textbf{Thrombotic MI} & \textbf{Non-Thromb. MI} \\
	\hline
	\textbf{sCAD}   &     13    &     0    &     2\\
	\textbf{Thrombotic MI}  &  0    &    10    &     1\\
	\textbf{Non-Thromb. MI} &   1    &     1   &     10
\end{tabular}
\end{frame}

\begin{frame}{LOO-CV estimation of error}
	\vspace{-15pt}
	Bayesian model with troponin:
	\vspace{4ex}
	
\begin{tabular}{l|ccc}
	& \emph{Predicted} & & \\
	\emph{Group}  &     \textbf{sCAD} & \textbf{Thrombotic MI} & \textbf{Non-Thromb. MI} \\
	\hline
	\textbf{sCAD}   &     13    &     0    &     2\\
	\textbf{Thrombotic MI}  &  0    &    10    &     1\\
	\textbf{Non-Thromb. MI} &   2  &   0 &    10
\end{tabular}
\end{frame}

\begin{frame}{LOO-CV estimation of error}
	\vspace{-15pt}
	Bayesian model with troponin:
	\vspace{2ex}
	\begin{center}
		\begin{tabular}{|c|c|cc|cc|}
			\hline
			& & \multicolumn{2}{|c|}{Sensitivity} & \multicolumn{2}{|c|}{Specificity} \\
			Model & Accuracy & Thromb. & Non-Thromb. &  Thromb. & Non-Thromb.  \\
			\hline
			M0 & 78.9\% & 81.8\% & 66.7\% & 92.6\% & 88.5\% \\
			M1 & 86.8\% & 90.9\% & 83.3\% & 96.3\% & 88.5\% \\
			M2 & 86.8\% & 90.9\% & 83.3\% & 100.0\% & 88.5\% \\
			\hline
		\end{tabular}
	\end{center}
\end{frame}

\section{Bayesian methods for compound identification}
\subsection{Priors from biochemical transformations}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=hide]
\end{frame}

\begin{frame}{Compound identification}
	\begin{columns}
		\begin{column}{0.45\textwidth}
			\vspace{-35pt}
			\begin{center}
				\includegraphics[scale=.275]{LCMS1b}
			\end{center}
		\end{column}
		\begin{column}{0.55\textwidth}
			\textbf{Features observed in MS:} \pause
			\begin{itemize}
				\item Mass-to-charge ratio (m/z) of the ions \pause
				\item Fragmentation pattern of either parent or fragment ions (if generated) \pause
				\item Isotopic distribution (e.g. the relative abundance of isotopes can be evaluated by comparing the ratio of 13C to 12C) \pause
				\item Elution times from chromatographic separation \pause
				\item Peak shape
			\end{itemize}
		\end{column}
	\end{columns}
\end{frame}

\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Priors from biochemical transformations}{Rogers, et al. (2009).}
	\vspace{-15pt}
	\begin{center}
		\includegraphics[scale=.575]{rogers1}
		
		Rogers, et al. (2009). doi:10.1093/bioinformatics/btn642.
	\end{center}
\end{frame}

\begin{frame}{Priors from biochemical transformations}{Rogers, et al. (2009).}
	\vspace{-12pt}
	\begin{itemize}
		\item Noise error model conditioned on an assignment:
		\begin{align*}
		\left. \frac{x_{m_i}}{x_{\ell_j}} \right| \{ I(m_i=\ell_j )=1 \} \sim N\left(1, \gamma^{-1}\right).
		\end{align*} \pause
		\item[]
		\item Conditional prior probability of an assignment:
		\begin{align*}
		p( I(m_i=\ell_j )=1|\mathbfcal{I},\delta) =  \frac{\beta_{ji}+\delta}{N\delta+\sum_{j'i} \beta_{j'i}}
		\end{align*}
	\end{itemize}
\end{frame}

\begin{frame}{Priors from biochemical transformations}{Rogers, et al. (2009).}
	\vspace{-12pt}
	\begin{itemize}
		\item The number of ways a compound can be ``made'':
		\begin{align*}
		\beta_{ji}=\textbf{W}_{j \cdot} \mathbfcal{I} \textbf{1} - \textbf{W}_{j \cdot} \mathbfcal{I}_{\cdot i} 
		\end{align*}
		\begin{itemize}
					\item $\textbf{W}$ as an indicator matrix that represents possible reactions
					\item $\delta$ is a hyperparameter
					\item $\beta_{ji}$ represents a count of the number of biochemical transformations from compound labels currently assigned to result in a new annotation \pause
					\item[]
		\end{itemize}
		\item Rogers et al. propose a Gibbs sampler for sampling from the posterior:
			\begin{align*}
				p( I(m_i=\ell_j )=1|\mathbfcal{I},x_m,\delta,\gamma) \propto N \left( \frac{x_{m_i}}{x_{\ell_j}} | 1,\gamma^{-1} \right) \frac{\beta_{ji}+\delta}{N\delta+\sum_{j'i} \beta_{j'i}}
			\end{align*}
	\end{itemize}
\end{frame}

\subsection{Formalization and integration of pathway databases}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Formalization and Pathway Databases}{Silva, et al. (2014)}
	\vspace{-12pt}
Silva, et al. formalized a comprehensive model:
			\begin{align*}
			p( \mathbfcal{I}|\textbf{Y}) & \propto p(\textbf{Y}| \mathbfcal{I}) p( \mathbfcal{I}) \\
			p(\mathbfcal{I}|\textbf{Y}) & \propto p_N(\textbf{Y}|\mathbfcal{I}) \cdot p_{rt}(\textbf{Y}|\mathbfcal{I}) \cdot p_{iso}(\textbf{Y}|\mathbfcal{I}) \cdot p( \mathbfcal{I}),
			\end{align*} 
			\begin{itemize}
				\item $p_N(\cdot)$ is an error model for the measurement of $m/z$
				\item $p_{rt}(\cdot)$ is a retention time / index error model
				\item $p_{iso}(\cdot)$ is an isotopic distribution error model
				\item $p( \mathbfcal{I})$ is the prior probability of compound label to mass feature assignments
			\end{itemize}
\end{frame}

\begin{frame}{Formalization and Pathway Databases}{Silva, et al. (2014)}
	\vspace{-12pt}
	\begin{itemize}
		\item The probability distribution of compound labels for a specific mass feature, conditioned on other assignments, is then:
		\begin{align*}
		p( I(m_i=\ell_j )| \mathbfcal{I}^{(-m_i)}) &\propto \\ p_N(x_{m_i}| I(m_i=\ell_j )) &\cdot p_{rt}(t_{m_i}|\mathbfcal{I}^{(-m_i)}) \cdot p_{iso}(\textbf{r}_{m_i}|\mathbfcal{I}^{(-m_i)}) \cdot p( \mathbfcal{I}^{(-m_i)}).
		\end{align*} \pause
		\item[]
		\item Also proposed a new m/z error model
		\begin{align*}
		p_N(x_{m_i}| I(m_i=\ell_j ),w) \propto I(x_{\ell_j}\in N_w(x_{m_i})) \left(1-\Phi\left(\frac{|x_{m_i}/x_{\ell_j}-1|-\mu_w}{\sigma_w}\right) \right),
		\end{align*}
		where $N_\varepsilon(x)$ is a neighborhood about the point $x$ with radius $\varepsilon$
	\end{itemize}
\end{frame}

\begin{frame}{Formalization and Pathway Databases}{Silva, et al. (2014)}
	\vspace{-12pt}
	\begin{center}
		\includegraphics[scale=.75]{Silva}
	\end{center}
\end{frame}

\subsection{Bayesian clustering of mass features}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=show/shaded/hide]
\end{frame}

\begin{frame}{Bayesian clustering of mass features}{Daly, et al. (2014).}
	\vspace{-12pt}
	\begin{center}
		\includegraphics[scale=1]{Daly1}
		
		Daly, et al. (2014). doi: 10.1093/bioinformatics/btu37
	\end{center}
\end{frame}

\begin{frame}{Bayesian clustering of mass features}{Daly, et al. (2014).}
	\vspace{-12pt}
	\begin{itemize}
		\item m/z measurement error:
			\begin{align*}
			p(x_n|v_{nkai}=1) = N(log(x_n)|log(y_{\phi_k ai}),\zeta^{-1}). 
			\end{align*} \pause
		\item Isotope profile error:
			\begin{align*}
			p(w_n|v_{nkai}=1)=N(\beta_{\phi_k ai} \lambda_{*},\kappa^{-1}+\beta^2_{\phi_k ai}\kappa^{-1}_{*}).
			\end{align*} \pause
		\item Retention time error:
			\begin{align*}
			p(r_n|v_{nkai}=1)=N(r_n|\mu_*,\delta_*^{-1}+\gamma^{-1}).
			\end{align*}
	\end{itemize}
\end{frame}

\begin{frame}{Bayesian clustering of mass features}{Daly, et al. (2014).}
	\vspace{-12pt}
	Conceptual overview of sampling routine:
	
	\begin{itemize}
		\item Initialize with random clustering
		\begin{itemize}
					\item Compute cluster posterior probabilities as well as new proposal cluster posterior probabilites
					\item[]
					\item If proposal is accepted then compute new metabolite, adduct, and isotope posterior probabilities
					\item[]
		\end{itemize}
		\item When convergence criteria is satisfied return assignments 
	\end{itemize}
\end{frame}

\section{Discussion}

\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=hide]
\end{frame}

\begin{frame}{Discussion}
	\vspace{-10.5pt}
	\begin{itemize}
		\item Metabolomics experiments / studies are characterized by $\downarrow n$ and $\uparrow p$ \pause
		\item[]
		\item There are many sources of extra-experimental knowledge that are relevant to metabolomics studies \pause
		\item[] 
		\item ``Extra-experimental prior beliefs'': \pause
		\begin{itemize}
			\item Structurally related metabolites are more likely to exhibit partial dependency than those that are not structurally related \pause
			\item Diagnostic models from metabolites should have moderated coefficient estimates \pause
			\item Adduct \& isotopic peaks in LC-MS data should co-elute, isotopic peaks should have predictable intensity ratios, and the presence of one metabolite should increase the likelihood of the presence of biochemically related metabolites
		\end{itemize}
	\end{itemize}
\end{frame}

\section{Acknowledgments}
\begin{frame}{Outline}
	\vspace{-10.5pt}
	\tableofcontents[currentsection,subsectionstyle=hide]
\end{frame}

\begin{frame}{Acknowledgments}
	\vspace{-5.5pt}
	\begin{center}
		\includegraphics[scale=.95]{../../../TempPics/Lab/Edits/DSC_0278-2}
	\end{center}
\end{frame}

\begin{frame}{Acknowledgments}
	\vspace{-5.5pt}
	\begin{center}
		\includegraphics[scale=.25]{fam}
	\end{center}
\end{frame}

\begin{frame}{Acknowledgments}
	\vspace{-5.5pt}
	\begin{center}
		\includegraphics[scale=.25]{sam}
	\end{center}
\end{frame}

\begin{frame}{Acknowledgments}
	\vspace{-15.5pt}
	\begin{center}
		\includegraphics[scale=.5]{../Metabolomics2018/SUSignatureHorizontal-2color}
	\end{center}
\end{frame}

\begin{frame}{Acknowledgments}
	\begin{columns}
		\begin{column}{0.5\textwidth}
			\textbf{Doctoral supervision}:
			\begin{itemize}
				\item Andrew DeFilippis, MD, MSc
				\item Shesh Rai, PhD
				\item[]
			\end{itemize}
			\textbf{Funding support}:
			\begin{itemize}
				\item NIH NIGMS 2P20GM103492-06 (Metabolomic Analysis of Atherothrombosis)
				\item Alpha Phi Foundation Heart-to-Heart Award
				\item NIH 1U24DK097154 (PI: Oliver Fiehn) Subproject Award
				\item[]
			\end{itemize}
		\end{column}
		\vspace{-25.5pt}
		\begin{column}{0.5\textwidth}
			\textbf{Doctoral committee }
			\begin{itemize}
				\item Eric Rouchka, DSc
				\item Juw Won Park, PhD
				\item[]
			\end{itemize}
			\textbf{Collaborators}:
			\begin{itemize}
				\item Joshua Mitchell 
				\item Hunter Moseley, PhD
				\item Samantha Carlisle, MS
				\item[]
			\end{itemize}
			\textbf{Lab}: Atherosclerosis/Atherothrombosis Research Laboratory (Twitter: @AtheroLab)
		\end{column}
	\end{columns}
\end{frame}

\end{document}